docs/http.rst

```diff
@@ -258,30 +261,32 @@ For the following call definitions:
 Allowed bridges are `gcm` (Google Cloud Messaging), `fcm` (Firebase Cloud
 Messaging), and `apns` (Apple Push Notification system)
 
-   :{token}: The bridge specific public exchange token
+   :{client_token}: The bridge specific public exchange token
 
-Each protocol requires a unique token that addresses the remote application.
+Each protocol may require a unique token that addresses the remote application
```

Comment:
I wonder if we should call this the UA instead of "remote application," here and everywhere. When we're talking about endpoints and the WebSocket protocol, we use "app" and "app server" to refer to the web app, but here, "application" refers to Fennec (the UA).